### DISCUSSION TITLE
Crashes Explorer on new Dev build of Windows 11 when adjusting volume or brightness

### DISCUSSION THREAD

**Discussion by IJuanTM** [valinet/ExplorerPatcher#843] [General]
Hi,

On the new Windows 11 Dev build they added new flyouts for volume and brightness, but when adjusting these it will crash the Explorer when EP is installed.

**Comment by valinet** [comment]
It's been reported already in a couple of threads. EP is not supported in Insider Preview builds. This is kind of expected since no one at Microsoft probably tests Windows 11 against EP. Contributions are, as always, welcome, I don't use Insider Preview builds daily.
